assets/onboarding/log-event.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Send log event.
+ *
+ * @param {string} eventName Event name.
+ * @param {Array} properties Event properties.
+ */
+export const logEvent = ( eventName, properties ) => {
+	window.sensei_log_event( eventName, properties );
+};
+
+/**
+ * Enable or disable event logging.
+ *
+ * @param {boolean} enabled Enabled state.
+ */
+logEvent.enable = ( enabled ) => {
+	window.sensei_event_logging.enabled = enabled;
```

Comment:
There is no page reload happening after the welcome step, where the user can change the tracking preference. This is needed so that changes takes effect immediately on the client side for the current page.
Without this, tracking events wouldn't be sent for the setup wizard if the user opts-in.

Reply:
Got it, thanks